keras/backend/mxnet_backend.py

```diff
@@ -1741,9 +1784,20 @@ def less_equal(x, y):
     if isinstance(y, KerasSymbol):
         y = y.symbol
         scalar = True
-    if scalar:
-        return KerasSymbol(mx.sym.Cast(x <= y, dtype='uint8'))
-    return KerasSymbol(mx.sym.Cast(mx.sym.broadcast_lesser_equal(lhs=x, rhs=y), dtype='uint8'))
+    if isinstance(x, mx.sym.Symbol) and isinstance(y, mx.sym.Symbol):
+        # use broadcasting to do element-wise comparison if both x and y are mxnet symbol
+        out = KerasSymbol(mx.sym.Cast(mx.sym.broadcast_lesser_equal(lhs=x, rhs=y), dtype='uint8'))
+    elif scalar:
+        # directly use '<=' operator for element-wise comparison
+        out = KerasSymbol(mx.sym.Cast(x <= y, dtype='uint8'))
```

Comment:
[minor] check style

Reply:
[adjust spaces around '=']

Reply:
@kalyc usually there is no space for optional params values, refer to any other operators.